src/errordialoghandler.cpp

```diff
@@ -29,7 +29,9 @@ ErrorDialogProperties::ErrorDialogProperties()
           m_modal(true),
           m_shouldQuit(false),
           m_type(DLG_NONE),
-          m_icon(QMessageBox::NoIcon) {
+          m_icon(QMessageBox::NoIcon),
+          m_defaultButton(QMessageBox::NoButton),
```

Comment:
Until now, the value was undefined and just taken from whatever was there in memory. Often, this is `0`, corresponding to `NoButton`, cf. <http://qt-project.org/doc/qt-4.8/qmessagebox.html>.